modules/installation-osp-updating-network-resources.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-osp-updating-network-resources_{context}"]
+= Updating network resources on {rh-openstack}
+
+Update the network resources that an {product-title} on {rh-openstack-first} installation on your own infrastructure requires. To save time, run supplied Ansible playbooks that generate security groups, networks, subnets, routers, and ports.
```

Comment:
I would remove the "To save time," part as it was already mentioned in the network section.